pallets/ddc-customers/src/lib.rs

```diff
@@ -511,6 +515,26 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// Mark existing bucket with specified bucket id as removed
+		///
+		/// Only an owner can remove a bucket
+		#[pallet::call_index(6)]
+		// Todo: add benchmarks
+		#[pallet::weight(T::WeightInfo::create_bucket())]
```

Comment:
the fn remove_bucket has weight of create_bucket.... please change

Reply:
Updated